conf/base.config

```diff
@@ -26,16 +26,16 @@ process {
     errorStrategy = { task.exitStatus in [143,137] ? 'retry' : 'ignore' }
   }
   withName: make_SILVA_132_16S_classifier {
-    cpus = { check_max (2 * task.attempt, 'cpus' ) }
-    memory = { check_max (35.GB * task.attempt, 'memory' ) }
+    cpus = { check_max (1 * task.attempt, 'cpus' ) }
+    memory = { check_max (40.GB * task.attempt, 'memory' ) }
     time = { check_max (12.h * task.attempt, 'time' ) }
   }
 
   //max memory seen yet: 63.GB
   withName: classifier {
-    cpus = { check_max (8 * task.attempt, 'cpus' ) }
-    memory = { check_max (28.GB * task.attempt, 'memory' ) }
-    time = { check_max (2.h * task.attempt, 'time' ) }
+    cpus = { check_max (20 * task.attempt, 'cpus' ) }
+    memory = { check_max (120.GB * task.attempt, 'memory' ) }
```

Comment:
120GB is too high in my opinion, this is not required and might lead to unnecessary long queuing or even problems when running on hardware that does not support such high values.

Reply:
This code is already increasingly with retries is it not?

What's wrong with allowing users to threshold the memory that is requested using the [`--max_memory` parameter](https://github.com/nf-core/ampliseq/blob/master/docs/usage.md#--max_memory)? This seems to be to be the most flexible approach - set the default to be the fastest possible for those who have the resources, but document that it is easy for people to limit this with the above command-line parameter if this proves to be a problem..

Reply:
For me, default settings should be sensible for most of the users. In my experience, 120Gb is not (queing increases, hardware lmitations). Additional settings should be used in the minority of the cases, and that is what 120Gb in this case is, again, in my opinion. Increasing from e.g. 60Gb to 120Gb is a minor increase in speed for very diverse samples (again, minority) but a hassle for many others.